### DISCUSSION TITLE
How to display a hierarchy of items?

### DISCUSSION THREAD

**Discussion by tbelmega** [cloudscape-design/components#280] [Q&A] [answered]
Hi friends,
I'm looking for an option to display an tree-like data structure, and make the items selectable.
To be specific, I want to display an AWS Organisation with its nested organizational units and account ids.
organization
```
|- ou-1
|    |- account-1
|    |- account-2
|- ou-2
     |- ou-3
     |    |- account-3
     |- ou-4
     |    |- account-4
     |    |- account-5
```

My first thought was to use the MultiSelect component with groups for organizational units, but MultiSelect only supports one level of grouping, while my use case requires multiple levels.
Another option is a table with selectable rows, but I don't see a good option to visualize that certain items in the table are children of other items, e.g. via indentation.

Is there a good way to do it with CloudScape, or do I need to build a custom component for that?

**Comment by connorlanigan** [comment, marked as answer]
Hello,

Cloudscape does not have a component for such a structure, so I recommend that you build a custom component.

Our [design tokens](https://cloudscape.design/foundation/visual-foundation/design-tokens/) might be useful during the implementation.
